app/controllers/moments_controller.rb

```diff
@@ -6,22 +6,32 @@ class MomentsController < ApplicationController
 
   before_action :set_moment, only: %i[show edit update destroy]
   before_action :load_viewers, only: %i[new edit create update]
+  respond_to :json, only: [:index]
 
   # GET /moments
   # GET /moments.json
+  # rubocop:disable MethodLength
   def index
-    if current_user
-      # +1 day buffer to ensure we include today as well
-      end_date = Date.current + 1.day
-      start_date = end_date - 1.week
-      @react_moments = current_user.moments
-                                   .group_by_period('day',
-                                                    :created_at,
-                                                    range: start_date..end_date)
-                                   .count
-    end
     page_collection('@moments', 'moment')
+    respond_to do |format|
+      format.json do
```

Comment:
I think we could move each of these respond_to blocks into a separate helpers for readability.